CenterForOpenScience/osf.io
# [ENG-2739] Upgrade webpack/ fix historyjs

## Purpose

This updates webpack to improve our js dependencies, it also fixes a bug in the Historyjs replacement PR.

## Changes

- update webpack and add resolution

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify the project and search pages load and work as expected.
- Verify

What are the areas of risk?

Any concerns/considerations/questions that development raised?

## Documentation

🐞 fix, no docs.

## Side Effects

None that I know of.


## Ticket

https://openscience.atlassian.net/browse/ENG-2739
